test/data/doubles_multiline_string.py

```diff
@@ -1,7 +1,7 @@
-""" This "should"
+s = """ This "should"
```

Comment:
Ah, you're right. `ast.get_docstring(ast.parse('"""foo"""'))` evaluates  to `'foo'`